### PR TITLE
Create a page for fitted country predictions.

### PR DESCRIPTION
**Pull request by miskosz**
The file `country_dashboard.py` needs a refactor, I did not do it because we will hopefully deprecate the file soon with The New Web™.

I think we should merge this only if:

- Memory is not an issue
- We want to provide users value (show predictions for recent dates) before we implement a proper solution (generate predictions with data import)

### CODE REVIEW

File: covid_graphs/covid_graphs/scripts/server.py
Comment (lukipuki) [inline covid_graphs/covid_graphs/scripts/server.py]: This is a strange endpoint name. We have `/all` and `/single`, and `/country` to me sounds like synonymous with `/single`. 
```suggestion
    @server.route("/covid19/predictions/automatic/")
```

File: covid_graphs/covid_graphs/scripts/country_dashboard.py
Comment (lukipuki) [inline covid_graphs/covid_graphs/scripts/country_dashboard.py:220]: It's fine doing it at the restart (it might become a problem with 100 countries though).

File: covid_graphs/covid_graphs/scripts/country_dashboard.py
Comment (lukipuki) [inline covid_graphs/covid_graphs/scripts/country_dashboard.py:108]: I feel like we've been talking about adding this field since 2018, way before COVID-19 :grinning: 
```suggestion
                            label=(event.creation_date).strftime("%B %d"),
```

**Review by lukipuki — APPROVED** [review]
I'm really motivated to publish some automated predictions soon, ideally before the weekend or on Saturday, so let's merge this even if it means we need to buy some server memory (it looks like we don't, though).

Reworking the web to the new version with a constantly changing backend and us working full-time jobs will take time (paging @rejdi ), so we'll have to live with `country_dashboard.py` for a while. I have actually started refactoring it in hopes of reducing memory footprint further - it looks like it didn't help but the code is nicer :smile: 

Before linking this from `index.html`, we should improve the labels (and probably also drop the very first prediction). Could be another PR.
Screenshot_20200428_001944

File: covid_graphs/covid_graphs/scripts/server.py
Comment (miskosz) [reply covid_graphs/covid_graphs/scripts/server.py]: What do you think about `daily`?

File: covid_graphs/covid_graphs/scripts/country_dashboard.py
Comment (miskosz) [reply covid_graphs/covid_graphs/scripts/country_dashboard.py:108]: Good idea, I'll do it. In a short next PR :)

File: covid_graphs/covid_graphs/scripts/server.py
Comment (lukipuki) [reply covid_graphs/covid_graphs/scripts/server.py]: Sounds good

**Pull request merged**
